src/assessments/semantics/test-steps/headers.tsx

```diff
@@ -70,7 +70,7 @@ const key = SemanticsTestStep.headers;
 
 export const Headers: Requirement = {
     key,
-    name: 'Headers',
+    name: 'Row/Column headers',
```

Comment:
Should probably change the variable/file names too (but maybe wait until LL and Fer confirms on "row/column" vs "table" wording)